javaprocessor/Dockerfile

```diff
@@ -16,11 +16,7 @@
 FROM ubuntu:xenial
 
 RUN \
- echo "deb http://repo.sawtooth.me/ubuntu/ci xenial universe" >> /etc/apt/sources.list \
- && apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 8AA7AF1F1091A5FD \
- && echo "deb http://repo.sawtooth.me/ubuntu/nightly xenial universe" >> /etc/apt/sources.list \
```

Comment:
@danintel Removed this line too, as all requirements are building through pom.xml now